static/js/subs.js

```diff
@@ -316,7 +308,7 @@ export function update_settings_for_subscribed(sub) {
 }
 
 export function show_active_stream_in_left_panel() {
-    const selected_row = get_hash_safe().split(/\//)[1];
+    const selected_row = hash_util.get_hash_section(hashchange.get_current_hash());
```

Comment:
This commit should presumably have some sort of commit message explaining why we expect the result to be the same.

Reply:
Okay added.